lineage/figures/figure8.py

```diff
@@ -1,55 +1,121 @@
 """
-File: figure7.py
-Purpose: Generates figure 7.
+File: figureS11.py
+Purpose: Generates figure S11.
```

Comment:
Isn't this figure 8?

Reply:
corrected. I replaced all of the figure S11 to figure 8 (everything was the same.)